devtools/client/framework/target-from-url.js

```diff
@@ -88,6 +91,27 @@ exports.targetFromURL = Task.async(function*(url) {
       }
       throw ex;
     }
+  } else if (type == "window") {
+    // Fetch target for a remote window actor
+    DebuggerServer.allowChromeProcess = true;
+    try {
+      id = parseInt(id, 10);
+      if (isNaN(id)) {
+        throw new Error("targetFromURL, " +
```

Comment:
nit: this all fits on one line? Also, can we please avoid doing white space like this (as it makes it hard to use beautifiers)? Just use 2 spaces everywhere.